contentcuration/contentcuration/migrations/0001_initial.py

```diff
@@ -1,11 +1,14 @@
 # -*- coding: utf-8 -*-
```

Comment:
@66eli77 @jayoshih We're now at a point where we should stop deleting and recreating migration files, as this means we need to destroy all data on the demo server in order to update it. If we keep the old migrations, and add new ones on top, then we can more seamlessly update the demo server while preserving data.